pkg/operator/ceph/cluster/watcher.go

```diff
@@ -565,7 +615,7 @@ func (c *clientCluster) unfenceAndDeleteNetworkFence(ctx context.Context, node c
 	}
 
 	err = wait.PollUntilContextTimeout(ctx, 2*time.Second, 60*time.Second, true, func(ctx context.Context) (bool, error) {
-		err = c.client.Get(ctx, types.NamespacedName{Name: node.Name, Namespace: cluster.Namespace}, networkFence)
+		err = c.client.Get(ctx, types.NamespacedName{Name: node.Name + "cephFS", Namespace: cluster.Namespace}, networkFence)
```

Comment:
Let's just keep it lower case, IMO the K8s resource naming is more consistent that way
```suggestion
		err = c.client.Get(ctx, types.NamespacedName{Name: node.Name + "-cephfs", Namespace: cluster.Namespace}, networkFence)
```